components/player.js

```diff
@@ -40,6 +62,7 @@ export default function Player () {
             <media-mute-button></media-mute-button>
             <div className="spacer" />
             {hasAirplay && <AirPlayButton onClick={() => muxVideoRef.current.webkitShowPlaybackTargetPicker()} />}
+            <google-cast-launcher></google-cast-launcher>
```

Comment:
For some reason, this `google-cast-launcher` is staying `display: none;` I would expect the Google Cast framework loading would make this element visible